content/page/liste-speaker.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Liste Speaker"
+date: 2022-11-30T12:39:45+01:00
+draft: false
+---
+
+# Speaker.ine.s confirmé.e.s pour la track onestla.tech aux ApiDays Paris 2022
+
+- [**Marion Agé**](https://twitter.com/marion_age) : cogérante et CTO de la coopérative Les-Tilleuls.coop.
+- **Cécile Staehle** : Coopératrice en charge du développement RH chez Troopers.
+- [**Antonio A. Casilli**](https://www.casilli.fr/) : Antonio A. CASILLI est professeur de sociologie à l’Institut
+Polytechnique de Paris, où il coordonne le groupe de recherche DiPLab (Digital Platform Labor). Il est parmi les
+fondateurs du réseau INDL (International Network on Digital Labor).  
+  En 2020 il a été conseiller éditorial de la série documentaire de France Télévisions 
+“Invisibles – Les travailleurs du clic“, basée sur ses recherches.  
+Parmi ses ouvrages :
+  - _En attendant les robots_ (Seuil,  2019/Points 2021) 
+  - _Qu'est-ce que le digital labor ?_ (INA, 2015; avec D. Cardon)
+  - _Les liaisons numériques_ (Seuil, 2010)
+- [**Hubert Guillaud**](https://twitter.com/hubertguillaud) : blogueur et journaliste.
+- [**Guillaume Champeau**](https://www.champeau.info/) : légal et affaires publiques chez Clever Cloud, fondateur de Numérama.
+- [**Marine Gandy**](https://twitter.com/mupsigraphy) : présidente de Drupal France, ingénieure chez Platform.sh, militante féministe.
+- [**Magali Milbergue**](https://twitter.com/MagaliMilbergue) : membre du bureau de l'AFUP, développeuse, formatrice, activiste et militante féministe.
+- [**Marcy Charollois**](https://twitter.com/MarcyChama) : membre de Duchess, consultante, militante féministe.
+- [**Emmanuelle Aboaf**](https://twitter.com/eaboaf_) : développeuse et spécialiste de l'accessibilité.
+- **Camille Gaquiere** : Camille officie dans les métiers du web depuis plus de 10 ans, et plus particulièrement comme 
```

Comment:
Peut-être mettre un lien vers le Linkedin de Camille Gaquière : https://www.linkedin.com/in/camillegaquiere/